indico/modules/events/editing/models/revisions.py

```diff
@@ -18,61 +17,37 @@
 from indico.util.string import format_repr
 
 
-class InitialRevisionState(RichIntEnum):
-    __titles__ = [None, _('New'), _('Ready for Review'), _('Needs Confirmation')]
-    __css_classes__ = [None, 'highlight', 'ready', 'warning']
-    #: A revision that has been submitted by the user but isn't exposed to editors yet
+class RevisionType(RichIntEnum):
+    __titles__ = [None, _('New'), _('Ready for Review'), _('Needs Confirmation'), _('Accepted'), _('Needs Changes'),
```

Comment:
Might be worth to add context to some of those strings e.g. `pgettext('revision', 'New')`. Otherwise this is difficult to translate in gendered languages.

Reply:
As far as I can tell these titles aren't ever displayed. Maybe it would be better to remove them altogether
(although for `EditableState` it might be worth adding context, yeah)

Reply:
for the editable strings, would it be ok to do something like
```python
def _(s):
    return pgettext('editable', s)
```
(just asking because i dont see that anywhere else in the code and it would get super wordy otherwise)

Reply:
no, that would break i18n string extraction